coalib/results/result_actions/IgnoreResultAction.py

```diff
@@ -78,40 +79,95 @@ def apply(self, result, original_file_dict, file_diff_dict, language: str,
 
         return file_diff_dict
 
-    def get_ignore_comment(self, origin, language):
+    def get_ignore_comment(self, language, spaces: str=''):
         r"""
-        Returns a string of Ignore Comment, depending on the language
+        Returns a tuple consisting of a formattable string of Ignore
```

Comment:
you can not change the return type of a function.  That is a breaking change, as unpacking it will cause an exception.
Do not do that.

If necessary, create a new function which works differently, and try to patch the old function so it sort-of does the 'best' thing, or just remove the old function as that provides a clean breakage for people to investigate why a function disappeared.